ConsoleGameLibrary.sln

```diff
@@ -1,30 +1,50 @@
 ﻿
-Microsoft Visual Studio Solution File, Format Version 11.00
-# Visual C# Express 2010
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Ok with that. VS2010 is a discontinued product, it can't even be installed (or more precisely, activated once installed) anymore.